src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,27 @@ title = "Releases"
 weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
+## v0.14.2
+
+This is a bugfix release:
+
+* Fix issues in Azure cloud prepare:
+  * Machine set name now changed to region + uuid and limited to 20 characters to prevent issues long cluster name.
```

Comment:
```suggestion
  * Machine set name convention was changed to region + uuid and is now limited to 20 characters to prevent issues seen with long cluster names.
```